www/_posts/2018-02-23-cordova-windows-6.0.0.md

```diff
@@ -0,0 +1,91 @@
+
+## Changes
+
+- **Windows 10 / UWP** builds are now default, meaning `cordova build windows` will now build a Windows 10 UWP app by default. Windows (Phone) 8.1 is still supported of course, just use `cordova build windows -- --appx=8.1-win`, `cordova build windows -- --appx=8.1-phone` or an equivalent configuration option.
+- You can now build apps with a current installation of **Visual Studio 2017** (`15.5.x` at the time of writing) without any additional configuration or hacks (like the environment variable `VSINSTALLDIR` that was required for 5.0.0).
+    * Note: [Visual Studio 2017 doesn't support Windows 8.1 apps any more](https://docs.microsoft.com/en-us/visualstudio/productinfo/vs2017-compatibility-vs#windows-store-and-windows-phone-apps), so you can only build these apps with Visual Studio 2015 installed.
+- New **ENV variable `MSBUILDDIR`** allows to directly configure the MSBuild Tools to be used to build the app. While `VSINSTALLDIR` always has been a hack that accidentally also worked to switch between different MSBuildTools versions, we now we this functionality explicit:  Just set the ENV var to a your desired MSBuild folder (e.g. `C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\MSBuild\15.0\Bin` or `C:\Program Files (x86)\MSBuild\14.0\bin\`) and it will be used to build your project. 
+    * If you have Visual Studio 2017 and Visual Studio 2015 installed at the same time, the normal logic would always choose Visual Studio 2017's MSBuild 15 - and fail on a Windows (Phone) 8.1 project. If you set the environment variable MSBuild 14, it can successfully build your 8.1 apps.
+    * If the supplied path is invalid or doesn't contain a working MSBuild, the normal MSBuild selection logic will be triggered
+- Fixes several bugs
+
+## Installation
+
+As usual this new version will be added as the default `cordova-windows` version only with the next release of Cordova CLI. Until then, please use these commands to remove and re-add the `windows` platform:
+
+```
+cordova platform rm windows
+cordova platform add windows@6.0.0
+```
+
+### Known limitations
+
+As it's often the case, the 6.0.0 release of cordova-windows is not perfect and has some known limitations:
+
+- If you only have Visual Studio 2017 installed on your machine, you can not build Windows (Phone) 8.1 apps.
+- "Windows 10 Phone emulator is currently not supported. If you want to deploy to emulator, use Visual Studio instead."
+- [Some `build`](https://github.com/apache/cordova-windows/issues/266) and [requirements error messages](https://github.com/apache/cordova-windows/issues/267) are a bit misleading
+
+## Feedback
+
+If you encounter problems with this new version, you can use the [GitHub issues of the `cordova-windows` repository](https://github.com/apache/cordova-windows/issues) to do so now. (`cordova-windows` is the first Apache Cordova repository on GitHub that enabled issue tracking. Yay!)
+
+## Open questions
```

Comment:
General: I am not convinced that this is the best way to accomplish what appears to be your underlying goal, which I think is to determine whether or not it is still worth the effort to support the extra Windows 8.1, Windows Phone 8.1, and Windows 10 Mobile platforms.

As I said below I think no extra effort to support Windows 10 Mobile if we keep the ARM target CPU build.

For Windows 8.1 & Windows Phone 8.1 I already proposed the following approach on a private Slack channel:
- Step 1: raise an issue on cordova-windows stating that Windows 8.1 & Windows Phone 8.1 are deprecated and scheduled for removal from the next release
- step 2: add notice to this blog post that support for Windows 8.1 & Windows Phone 8.1 is now deprecated and scheduled to be removed from the next major release (7.0.0), along with a link to the issue for feedback and discussion

This would give the user community a chance to take action and speak up if support for Windows 8.1 / Windows Phone 8.1 is really needed, along with discussion of possible alternative solutions if needed.

Reply:
GitHub issues are not for discussion, but besides that I agree and reformulated that whole (unfinished) paragraph to something more along those lines.